src/components/TopNav/TopNav.scss

```diff
@@ -43,6 +43,16 @@ $top-nav-height: 60px;
     }
   }
 
+  &__left {
+    align-items: center;
+    background-color: var(--color-bg);
+    display: flex;
+    flex: 1;
```

Comment:
Do you need the `flex: 1` here? seem like not affecting anything

Reply:
I added `flex: 1` so that the entire drop menu would be covered until it gets below the header. Currently the logo appears to hide the menu as the menu drops, but only because the text content of the current menu items is not as wide as the logo.

Example: (change 'Join the Community!' to 'Join the Community! It is amazing!')

With `flex: 1` the menu item is hidden until below `.TopNav__left`
![Screen Shot 2020-12-15 at 8 33 19 AM](https://user-images.githubusercontent.com/12941979/102228755-aa76a300-3eb0-11eb-93cf-d6fc3839726a.png)

Without `flex: 1` the menu item will appear to the right of the logo on its way down.
![Screen Shot 2020-12-15 at 8 34 37 AM](https://user-images.githubusercontent.com/12941979/102228814-bbbfaf80-3eb0-11eb-8196-497896e018d2.png)

Technically this doesn't directly effect "blocking logo and icon", but I saw it as a future-oriented improvement.